src/style.css

```diff
@@ -0,0 +1,83 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  max-width: 1190px;
+}
+
+.block {
+  flex-grow: 1;
+  height: 300px;
+  min-width: 300px;
+
+  text-align: center;
+  line-height: 300px;
```

Comment:
use `display:flex`, `align-items:center` and `justify-content:center` for centering the text in blocks